dialogflow/spec/spec_helper.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def get_entity_type_ids project_id:, display_name:
+  # project_id = "Your Google Cloud project ID"
+  # display_name = "Existing Display Name"
+  
+  require "google/cloud/dialogflow"
```

Comment:
Move the `require` to the top of the file.

`require` should never be inside of a class/method/module (unless you have a very good reason)

^--- we put them into our snippet methods so the require will be inside of region tag blocks

Also, please remove the placeholder variable comments, these are part of the snippet code rubric and not idiomatic Ruby

```ruby
# LICENSE

require "google/cloud/dialogflow"

def get_entity_type_ids project_id, display_name
  client       = Google::Cloud::Dialogflow::EntityTypes.new
  parent       = client.class.project_agent_path project_id
  entity_types = entity_types_client.list_entity_types parent

  entity_types.select { |et| et.display_name == display_name }.
                  map { |et| et.name.split("/").last }
end

# ...
```

Reply:
done.